roles/openshift_azure/filter_plugins/azure_filters.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# pylint: disable=too-many-lines
+"""
+Custom filters for use in openshift_azure role.
+"""
+
+import json
+import urllib
+import urlparse
+from dateutil.parser import parse as date_parse
+from datetime import datetime, timedelta
+
+from ansible import errors
+from collections import Mapping
+
+def openshift_azure_convert_list_to_json(sas_data_list):
+    """ Receives a list of stdout sas data objects
+        Ex: "{{ sas_urls | openshift_azure_convert_list_to_json }}"
+    """
+    if not sas_data_list:
+        raise errors.AnsibleFilterError("|failed expects sas_data to be set")
+
+    if not isinstance(sas_data_list, list):
+        raise errors.AnsibleFilterError("|openshift_azure_convert_list_to_json failed expects to filter on a list object")
+
+    results = []
+    for res in sas_data_list:
+        results.append(json.loads(res))
+
+    return results
+
+
+def openshift_azure_create_sas_url(sas_data):
```

Comment:
doesn't seem actually to be being used?

Reply:
Commented out.  I left this here so we have documentation and in case we need to revive it at some point since the generated URLs do not work out-of-the-box.